pkg/pb/nodestats.proto

```diff
@@ -10,11 +10,22 @@ import "gogo.proto";
 import "google/protobuf/timestamp.proto";
 
 service NodeStats {
+    rpc AuditCheck(AuditCheckRequest) returns (AuditCheckResponse);
     rpc UptimeCheck(UptimeCheckRequest) returns (UptimeCheckResponse);
```

Comment:
Is there a reason we are using two different checks for this rather than one to get full information?